charts/fybrik/files/taxonomy/data_catalog_response.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Catalog Response",
```

Comment:
response should also have operation type, and the content would be for the type.
Can you generalize? 
i also suggest moving the name, type and asset to common definition as in the future they will be used by operationtype "create"/"update" so we should architecture for future use.
Even as we start only with "get"/"read".

Reply:
Updated the base taxonomy to have the operationType enums. Please review. Thanks. 
Also moved name, type and asset to common definition under the definitions so that it can be reused later. I hope this was your suggested change. Please review. Thanks. @ronenkat

Reply:
I disagree about the need to echo request data in the response. If it's in the request we don't need it in the response.

Reply:
removed the field from the response object. Please check. Thanks.